docs/devenv_rhel8.md

```diff
@@ -245,3 +244,35 @@ go tool pprof heap.pprof
 ```
 
 To view all the available profiles, run `oc get --raw /debug/pprof`.
+
+## Troubleshooting
+### No valid repository ID for fast-datapath
+If you encounter following error message while enabling fast-datapath repo:
+
+```
+# sudo subscription-manager repos \
```

Comment:
Let's remove the command and leave the message alone. In general, the same problem may occur for rhocp channel enablement

Reply:
done.